packages/odyssey-design-tokens/src/border.json

```diff
@@ -1,14 +1,28 @@
 {
   "border": {
-    "width": {
-      "base": { "value": "1px" }
-    },
-    "style": {
-      "base": { "value": "solid" }
+    "color": {
+      "ui": { "value": "{palette.danger.light.value}" },
+      "display": { "value": "{hue.neutral.200.value}" },
+      "disabled": { "value": "{hue.neutral.200.value}" },
+      "danger": {
+        "light": { "value": "{palette.danger.light.value}" },
+        "base": { "value": "{palette.danger.main.value}" },
```

Comment:
We've been using `base`, but only because I made an arbitrary call a few years ago. MUI uses `main` instead, which we've mapped some color tokens to. `main` might make more sense, since we aren't actually modifying a "base" token.

Any thoughts here @gretchen-okta?

Reply:
Confirmed this change is 💯